.github/workflows/run-tests.yml

```diff
@@ -416,11 +416,18 @@ jobs:
           mkdir -p /tmp/minio_tests/test-bucket && chmod -R 777 /tmp/minio_tests
           docker-compose -f .docker/$DOCKERFILE run qgis-deps /root/QGIS/.docker/docker-qgis-test.sh $TEST_BATCH
 
+      - name: Save PR number to test report
+        if: ${{ failure() }}
+        env:
+          PR_NUMBER: ${{ github.event.number }}
+        run: |
+          echo $PR_NUMBER > qgis_test_report/pr_number
```

Comment:
@nyalldawson, it looks like this command always fails (when there is a test failure) with "permission denied" error:
```
Run echo $PR_NUMBER > qgis_test_report/pr_number
  echo $PR_NUMBER > qgis_test_report/pr_number
  shell: /usr/bin/bash -e {0}
  env:
    DOCKER_USERNAME: 
    DOCKER_PASSWORD: 
    QGIS_WORKSPACE: /home/runner/work/QGIS/QGIS
    CTEST_BUILD_NAME: PR55[2](https://github.com/qgis/QGIS/actions/runs/6843501155/job/18608800408?pr=55264#step:14:2)64_40efa27261fa9511a[3](https://github.com/qgis/QGIS/actions/runs/6843501155/job/18608800408?pr=55264#step:14:3)88730af3bed19563c9682f_ALL_BUT_PROVIDERS
    QT_VERSION: 5
    PR_NUMBER: 5526[4](https://github.com/qgis/QGIS/actions/runs/6843501155/job/18608800408?pr=55264#step:14:4)
/home/runner/work/_temp/f6b1d229-1adc-431a-a7e9-e61[5](https://github.com/qgis/QGIS/actions/runs/6843501155/job/18608800408?pr=55264#step:14:5)d98db7[6](https://github.com/qgis/QGIS/actions/runs/6843501155/job/18608800408?pr=55264#step:14:6)c.sh: line 1: qgis_test_report/pr_number: Permission denied
Error: Process completed with exit code
```

Reply:
@agiudiceandrea I'm working on it - see https://github.com/qgis/QGIS/pull/54919